packages/zoe/src/contractFacet/seat.js

```diff
@@ -73,6 +73,7 @@ export const makeZcfSeatAdminKit = (
       if (currentAllocation[keyword] !== undefined) {
         return currentAllocation[keyword];
       }
+      assert(brand, `A brand must be supplied if the keyword may not exist`);
```

Comment:
```suggestion
      assert(brand, `A brand must be supplied when the keyword is not defined`);
```
"may not" seems wrong. It's only required (by the implementation) when the keyword is not found. As advice to callers, I might say "when you're not sure that the keyword will be present, you should include the brand", but that's about their state of mind, not what is enforced.

Also, "exist" seems like the wrong verb. How do you feel about "is not present", "is not found" or "is not defined"?

Reply:
This new error message looks good to me. thanks!